docs/contributing.md

```diff
@@ -141,11 +141,6 @@ Indicate a breaking change by placing an `!` between the type name and the colon
 feat!: add a breaking feature
 ```
 
-or
-
-```
-chore!: drop support for Node.js v12
```

Comment:
I thought this example might cause confusion with the definition of `chore` - dropping support for Node.js v12 could hypothetically impact end users of the library, e.g. for consumers that still use Node v12.

Is there another type that covers this sort of work or should we revise the definition of `chore` accordingly?

Reply:
Interesting point! I agree I can't untangle the definition of chore ("any changes that don't impact end users of the library") which could also be a breaking change. Would be interested to hear what Rowan thinks :)

Reply:
Ah yeah I see what you mean 🤔 I think redefine `chore` because dropping Node.js version support does technically come under that heading (it's not a feature, it's not a fix, and it's not documentation).

Shall we go with `repo maintenance and support tasks`? I feel like maybe that covers it

Reply:
`repo maintenance and support tasks` works for me 😄

Reply:
Updated in [this commit](https://github.com/Financial-Times/dotcom-reliability-kit/commit/eab86bde7703f0245bc39c027d0f04dac52187f8).